source/write-data-to-mongo.txt

```diff
@@ -0,0 +1,162 @@
+.. _java-rs-write:
+
+=====================
+Write Data to MongoDB
+=====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :description: Learn how to use the {+driver-short+} to write data to MongoDB.
+   :keywords: usage examples, save, crud, create, code example
+
+.. TODO: change toctree links when other pages are merged in.
+.. .. toctree::
+..    :titlesonly:
+..    :maxdepth: 1
+
+..    /write/insert
+..    /write/update
+..    /write/replace
+..    /write/delete
+..    /write/bulk-write
+..    /write/gridfs
+
+Overview
+--------
+
+On this page, you can see copyable code examples that show common
+methods to write data to MongoDB with the {+driver-short+}.
```

Comment:
S: phrasing feels a little awkward. side question: what makes these methods common?

```suggestion
This page contains copyable code examples of
{+driver-short+} methods that you can use to
write data to MongoDB.
```

Reply:
I think they're just the main methods that people might use, but I agree that it feels a little prescriptive to call them common